api-reference/10 UI Components/dxForm/3 Methods/reset(editorsData).md

```diff
@@ -3,10 +3,142 @@ id: dxForm.reset(editorsData)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Resets the editors values to the values passed as an argument. Resets all other values to initial values.
```

Comment:
I rewrote this paragraph, because right now the sentences are not linked. If you read the first, you don't anticipate that a second will come. You think that ALL editors are already updated. But then you read about OTHER editors. I wanted to make sure that the reader can easily follow our thoughts - from the start they understand that the argument may or may not contain a value for a specific editor. 

```suggestion
Resets editor values. Every editor attempts to obtain its new value from the argument list. If an editor is not included in the list, that editor resets to its initial value.
```